build.gradle

```diff
@@ -2,6 +2,8 @@ plugins {
     id 'java'
     id 'application'
     id 'com.bmuschko.docker-remote-api' version '4.0.3'
+    id 'idea'
+    id 'net.ltgt.apt-idea' version "0.20"
```

Comment:
now i remember why these were here - otherwise intellij can't see the generated CRD classes